bitcoin/src/blockdata/weight.rs

```diff
@@ -31,30 +31,38 @@ impl Weight {
     /// Maximum possible value.
     pub const MAX: Weight = Weight(u64::MAX);
 
+    /// The factor that non-witness serialization data is multiplied by during weight calculation.
+    pub const WITNESS_SCALE_FACTOR: u64 = crate::blockdata::constants::WITNESS_SCALE_FACTOR as u64;
```

Comment:
This is a bit weird having it in two places, perhaps add a code comment saying something like `// We define this here as well so we don't have to cast all over the place.`

Reply:
I was actually thinking maybe `WITNESS_SCALE_FACTOR` could be removed from `constants` and placed here?  Although that will probably be a large change so I did this as an in-between.

Reply:
I'm going to mark this as resolved for now unless there's further commentary.